crux-http-server/src/crux/http_server.clj

```diff
@@ -199,6 +203,18 @@
     (-> (success-response (.entity db eid))
         (add-last-modified tx-time))))
 
+(defn- entity-state [^ICruxAPI crux-node request]
+  (let [[_ encoded-eid] (re-find #"^/_entity/(.+)$" (req/path-info request))]
```

Comment:
I'd say the time's come to start with a proper routing library - we've outgrown manually parsing out each individual value

Reply:
as discussed, bringing this in as a separate change